src/apps/concepts/pages/ViewConceptPage.tsx

```diff
@@ -84,13 +85,7 @@ const ViewConceptPage: React.FC<Props> = ({
           />
         </Grid>
         {!showEditButton ? null : (
-          <Link replace to={`${url}edit/?linkedDictionary=${linkedDictionary}`}>
-            <Tooltip title="Edit this concept">
-              <Fab color="primary" className="fab">
-                <EditIcon />
-              </Fab>
-            </Tooltip>
-          </Link>
+            <EditButton url={`${url}edit/?linkedDictionary=${linkedDictionary}`} title="Edit this concept"/>
```

Comment:
Typo: `c` should be capital in `concept`. For Dictionary and Source, D and S are capital resp.